src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -385,8 +392,156 @@ public void putOrdersById(TestContext ctx) throws Exception {
+     .contentType(APPLICATION_JSON)
+    .body("{}")
+   .post(rootPath+INVALID_LANG)
+     .then()
+       .statusCode(400)
+       .body(containsString("'lang' parameter is incorrect. parameter value {english} is not valid: must match \"[a-zA-Z]{2}\""));
+     
+  }
+  
+  @Test
+  public void testValidationOnGet(TestContext ctx) throws Exception {
+    logger.info("=== Test validation Annotation on GET API ===");
+
+    logger.info("=== Test validation on invalid offset query parameter ===");
+   RestAssured
+      .with()
+        .header(X_OKAPI_URL)
+        .header(X_OKAPI_TENANT)
+      .get(rootPath+INVALID_OFFSET)
+        .then()
+          .statusCode(400)
+          .body(containsString("'offset' parameter is incorrect. parameter value {-1} is not valid: must be greater than or equal to 0"));
+   
+   logger.info("=== Test validation on invalid lang query parameter ===");
+   RestAssured
+   .with()
+     .header(X_OKAPI_URL)
+     .header(X_OKAPI_TENANT)
+     .contentType(APPLICATION_JSON)
+   .get(rootPath+INVALID_LANG)
+     .then()
+       .statusCode(400)
+       .body(containsString("'lang' parameter is incorrect. parameter value {english} is not valid: must match \"[a-zA-Z]{2}\""));
+   
+   logger.info("=== Test validation on invalid limit query parameter ===");
+   RestAssured
+   .with()
+     .header(X_OKAPI_URL)
+     .header(X_OKAPI_TENANT)
+   .get(rootPath+INVALID_LIMIT)
+     .then()
+       .statusCode(400)
+       .body(containsString("'limit' parameter is incorrect. parameter value {-1} is not valid: must be greater than or equal to 0"));
+     
+  }
+  
+  @Test
+  public void testValidationOnGetById(TestContext ctx) throws Exception {
+    logger.info("=== Test validation Annotation on GET ORDER BY ID API ===");
+
+   
+   
+   logger.info("=== Test validation on invalid lang query parameter ===");
+   RestAssured
+   .with()
+     .header(X_OKAPI_URL)
+     .header(X_OKAPI_TENANT)
+     .contentType(APPLICATION_JSON)
+   .get(rootPath+INVALID_LANG)
+     .then()
+       .statusCode(400)
+       .body(containsString("'lang' parameter is incorrect. parameter value {english} is not valid: must match \"[a-zA-Z]{2}\""));
+
+     
   }
   
+  @Test
+  public void testValidationDelete(TestContext ctx) throws Exception {
+    logger.info("=== Test validation Annotation on DELETE API ===");
+
+   
+   
+   logger.info("=== Test validation on invalid lang query parameter ===");
+   RestAssured
+   .with()
+     .header(X_OKAPI_URL)
+     .header(X_OKAPI_TENANT)
+     .contentType(APPLICATION_JSON)
+   .get(rootPath+INVALID_LANG)
+     .then()
+       .statusCode(400)
+       .body(containsString("'lang' parameter is incorrect. parameter value {english} is not valid: must match \"[a-zA-Z]{2}\""));
+
+     
+  }
+  
+  @Test
+  public void testValidationOnPut() throws Exception {
+    logger.info("=== Test validation Annotation on PUT API ===");
+
+   logger.info("=== Test validation on empty body ===");
```

Comment:
You are testing for an empty body`{}` but not passing an empty body? I think you have done it correctly on line 524 but not here.

Reply:
Thank you, corrected the info, the test was to be tested with no body sent on a post event,